.github/workflows/required.yml

```diff
@@ -0,0 +1,21 @@
+name: Required Tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
You can use the `container: <image>` name to avoid starting our own containers

Ref: [PKI build job](https://github.com/dogtagpki/pki/blob/master/.github/workflows/required-tests.yml#L16)